bciers/apps/registration/tests/components/operations/registration/FacilityInformationForm.test.tsx

```diff
@@ -435,4 +435,43 @@ describe("the FacilityInformationForm component", () => {
 
     expect(streetAddress).toHaveValue("123 Test St");
   });
+
+  it("should direct the user to the next page of the form on successful submit", async () => {
+    window = Object.create(window);
+    const origin = "http://localhost:3000";
+    const path = `/registration/register-an-operation/002d5a9e-32a6-4191-938c-2c02bfec592d`;
+    Object.defineProperty(window, "location", {
+      value: {
+        href: `${origin}${path}/2`,
+        origin,
+      },
+      writable: true,
+    });
```

Comment:
I ran into difficulties spying on `window.location.href` specifically (window.location.get) seems to work fine so I ended up with this.

Reply:
Wow, that's a lot more complicated than spying on the router, thanks for figuring it out